### ISSUE TITLE
Boostnote not rendering notes correctly

### ISSUE THREAD

**Issue by federicochiesa** [BoostIO/Boostnote#3319]
# Current behavior

When I get out of editing mode(and sometimes even in editing mode), the note doesn't render correctly, or sometimes doesn't render at all, as you can see [here](https://youtu.be/gu3WGdixgF4), where it cuts out part of the note, or [here](https://youtu.be/uTW8K2KWi-k), where it renders wrong not only the editing mode(wrong highlighting after a split second when I scroll down), but the note doesn't render at all.

# Expected behavior

When I get out of the editor I should be able to see my note correctly.

# Steps to reproduce

1. Open a note
2. Start editing
3. Get out of editing mode

# Environment

- Version : 0.13
- OS Version and name : macOS 10.14.6

**Comment by duguanyue** [comment]
I have the same issue since updating to v0.13.

**Comment by Flexo013** [comment]
Duplicate of #3289

**Comment by Flexo013** [comment]
The first bug you mention is indeed a duplicate of #3289.

The second bug however is something that doesn't have an issue here yet, so I'm reopening this one. I have run into this same bug as well in one of my larger notes, but I haven't been able to narrow down what causes the issue.

All I know is that it occurs after a certain point in the document (in my case a level 3 heading). If the cursor is beyond this point then the entire note won't render. If the cursor is before this point then nothing will go wrong.

**Comment by federicochiesa** [comment]
You're right! I just have to set the cursor in the top line and I can see the note perfectly. At least now I have a temporary fix to see my note, since what I did before was switch to another note and switch back.

**Comment by Flexo013** [comment]
@federicochiesa Could you try placing your cursor on a few different points in the note to see at what point the note stops rendering? Maybe that will help in narrowing down the cause.

**Comment by federicochiesa** [comment]
It seems as if the loewr down I put the cursor, the more the rendered note frame shrinks: if I put it on the top line, the note is fully visible, and as I go down line by line I can see less and less, until I can see nothing

**Comment by Flexo013** [comment]
Hmm, maybe this is related to the scrolling bug #3289. Maybe the fix for that also fixes this?
